app/frontend/store.ts

```diff
@@ -2,7 +2,7 @@ import devtools from './libs/unistore/devtools'
 import {ADALITE_CONFIG} from './config'
 import {initialState} from './state'
 
-const createDefaultStore = require('unistore').default
+const createDefaultStore = require('./libs/unistore').default
```

Comment:
why is this change needed?

Reply:
without that I'm getting Module Not Found error, apparently when running tests I need the full path.